pkg/controllers/clusterthrottle_controller.go

```diff
@@ -378,27 +378,86 @@ func (c *ClusterThrottleController) UnReserveOnClusterThrottle(pod *corev1.Pod,
 func (c *ClusterThrottleController) CheckThrottled(
 	pod *corev1.Pod,
 	isThrottledOnEqual bool,
+	groupNameAnnotation string,
```

Comment:
Seems that PodGroup CRD groups pods using labels, not an annotations.
https://github.com/kubernetes-sigs/scheduler-plugins/tree/master/pkg/coscheduling#podgroup

Reply:
PodGroup is just an abstract name, and is not intended to be fully coscheduling.
However, I think there should be support for not only annotation but also labels. If users request it, we will implement it.

Currently, I'm planning to use this feature for our gang's annotations.
https://github.com/pfnet/scheduler-plugins/tree/master/plugins/gang#usage